flow/FastAlloc.cpp

```diff
@@ -156,7 +156,7 @@ void recordAllocation( void *ptr, size_t size ) {
 
 		uint32_t a = 0;
 		if( nptrs > 0 ) {
-			a = crc32c_append( 0xfdbeefdb, buffer, nptrs * sizeof(void *));
+			a = crc32c_append( 0xfdbeefdb, reinterpret_cast<uint8_t*>(buffer), nptrs * sizeof(void *));
```

Comment:
I think you can use a `static_cast` here, which is probably preferable. A quick look shows this is what we do in KeyValueStoreSQLite.actor.cpp.

```suggestion
			a = crc32c_append( 0xfdbeefdb, static_cast<uint8_t*>(buffer), nptrs * sizeof(void *));
```

Reply:
Unfortunately static cast won't work here. In order to perform the pointer to numerical values cast we need reinterpret_cast here. I tried static_cast and it won't compile, which is right since there is no implicit or user defined conversion between void** and const uint8_t*.

Reply:
Oh right, because buffer is a double pointer.